test/unit/contract/DPContractFactory.spec.js

```diff
@@ -60,24 +60,24 @@ describe('DPContractFactory', () => {
 
       const result = factory.createFromObject(rawDPContract);
 
-      expect(result).to.be.equal(dpContract);
+      expect(result).to.equal(dpContract);
 
-      expect(validateDPContractMock).to.be.calledOnceWith(rawDPContract);
+      expect(validateDPContractMock).to.have.been.calledOnceWith(rawDPContract);
 
-      expect(createDPContractMock).to.be.calledOnceWith(rawDPContract);
+      expect(createDPContractMock).to.have.been.calledOnceWith(rawDPContract);
     });
 
     it('should return new DPContract without validation if "skipValidation" option is passed', () => {
       const result = factory.createFromObject(rawDPContract, { skipValidation: true });
 
-      expect(result).to.be.equal(dpContract);
+      expect(result).to.equal(dpContract);
 
-      expect(validateDPContractMock).not.to.be.called();
+      expect(validateDPContractMock).to.have.not.been.called();
 
-      expect(createDPContractMock).to.be.calledOnceWith(rawDPContract);
+      expect(createDPContractMock).to.have.been.calledOnceWith(rawDPContract);
     });
 
-    it('should throw error if passed object is not valid', () => {
+    it('should throw an error if object passed was not valid', () => {
```

Comment:
i'd put passed before object. you can also use 'is' instead of 'was' but I don't think that matters as much.

'should throw an error if passed object is not valid'